common/dbconnector.cpp

```diff
@@ -186,8 +186,9 @@ void SonicDBConfig::initialize(const string &file)
 
     if (m_init)
     {
-        SWSS_LOG_ERROR("SonicDBConfig already initialized");
-        throw runtime_error("SonicDBConfig already initialized");
```

Comment:
Community will have a problem with this change. you can instead use reset()

Eg:
```
>>> SonicDBConfig::initializeGlobalConfig()
>>> SonicDBConfig.reset()
>>> SonicDBConfig.load_sonic_db_config('/var/run/redisdpu0/sonic-db/database_config.json')
>>> SonicDBConfig.getDbPort('CONFIG_DB')
6381
>>> SonicDBConfig.getDbHostname('CONFIG_DB')
'169.254.200.254'
```